libs/product/src/drivers/magento/transforms/bundled-product-transformers.ts

```diff
@@ -35,9 +35,31 @@ function transformMagentoBundledProductItemOption(option: MagentoBundledProductI
 	return {
 		id: option.id.toString(),
 		name: option.label,
-		price: option.price,
+		price: getPrice(option.product),
+		discount: getDiscount(option.product),
 		quantity: option.quantity,
 		is_default: option.is_default,
 		in_stock: option.product.stock_status === MagentoProductStockStatusEnum.InStock
 	}
 }
+
+/**
+ * A function for null checking an object.
+ */
+function getPrice(product: MagentoProduct): number {
+	return product.price_range && 
+		product.price_range.maximum_price && 
+		product.price_range.maximum_price.regular_price && 
+		product.price_range.maximum_price.regular_price.value !== null
+	? product.price_range.maximum_price.regular_price.value : null;
+}
+
+function getDiscount(product: MagentoProduct): DaffProductDiscount {
+	return product.price_range && 
+		product.price_range.maximum_price && 
+		product.price_range.maximum_price.discount 
+		? { 
+			amount: product.price_range.maximum_price.discount.amount_off,
+			percent: product.price_range.maximum_price.discount.percent_off
+		} : { amount: null, percent: null }
```

Comment:
Do you think these values should be initialized to 0? I feel like they should, but that is also probably handled in the state layer anyway.

Reply:
I think it should be null, since it is passed to daffodil as null and not 0